msteams-platform/concepts/build-and-test/analyze-your-apps-usage-in-developer-portal.md

```diff
@@ -114,157 +117,159 @@ You can view usage for your [third-party app](../../promote-app-adoption.md#acce
 1. Select the required app from the **Apps** page.
 1. Select **Analytics** under the **Overview** section.
```

Comment:
Check this comment: https://github.com/MicrosoftDocs/msteams-docs/pull/12737#discussion_r2083924043